ksqldb-engine/src/test/java/io/confluent/ksql/integration/EndToEndIntegrationTest.java

```diff
@@ -110,7 +110,7 @@
   @Parameterized.Parameters(name = "{0}")
   public static Collection<Boolean> data() {
     return Arrays.asList(
-        true, false
```

Comment:
I have a ticket to remember to clean this up later https://confluentinc.atlassian.net/browse/KCI-1240